src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java

```diff
@@ -353,56 +351,38 @@ private OutputAlleleSubset(final List<Allele> alleles, final List<Integer> mleCo
      * @return information about the alternative allele subsetting {@code null}.
      */
     private OutputAlleleSubset calculateOutputAlleleSubset(final AFCalculationResult afCalculationResult, final VariantContext vc) {
```

Comment:
@davidbenjamin Here is the motivation for the change: https://github.com/broadinstitute/gsa-unstable/issues/1188.